docs/UI/CancelInteraction/index.md

```diff
@@ -0,0 +1,76 @@
+## CancelInteraction
+
+Type
+: Function
+
+Sender
+: Mobile
```

Comment:
```suggestion
: SDL
```

These guides mainly describe the messaging between Core and the HMI, so Core is the sender from this perspective.